about.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="src/assets/stylesheets/style.css">
+  <script src="https://kit.fontawesome.com/e2597287a1.js" crossorigin="anonymous"></script>
+  <title>Minute Clinic Conference</title>
+</head>
+<body>
+  <div class="app-cont">
+    <header>
+      <div class="about-cont">
+        <div class="sub-header">
+          <div class="sub-header-flex">
+            <ul>
+              <li class="sub-header-icon"><i class="fa-brands fa-facebook-f"></i></li>
+              <li class="sub-header-icon"><i class="fa-brands fa-twitter"></i></li>
+              <li>English</li>
+              <li>My Page</li>
+              <li>Logout</li>
+            </ul>
```

Comment:
- [ ] Since they are a list of links, Kindly consider grabbing them inside the anchor tag.
`Hint: You can say `
`<li><a href=""></a></li>`

Reply:
Corrected!